src/components/common/Navbar.jsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import Hamburger from "./hamburger";
+import { MdKeyboardArrowDown } from "react-icons/md";
+import { FaDiscord } from "react-icons/fa";
+
+const Navbar = () => {
+  const list = [
+    {
+      name: "Org",
+      link: "/org",
+      subMenu: [
+        { name: "About", link: "/about" },
+        {
+          name: "Contact us ",
+          link: "/bitspace/#contact_us",
+        },
+      ],
+    },
+    {
+      name: "Bitcoin Career",
+      link: "/#bitcoin_career",
+      subMenu: [
+        { name: "Study cohorts", link: "/cohorts" },
+        { name: "Fellowship", link: "/fellowship" },
+      ],
+    },
+    {
+      name: "IRL Funzzz",
+      link: "/#irl_funzzz",
+      subMenu: [
+        { name: "Bitspace", link: "/bitspace" },
+        { name: "BITDEVS", link: "/meetup" },
```

Comment:
This config seems to be same as that in hamburger. can we use the same one instead of rewriting

Reply:
i was making two seperate prs, so used two lists. will combine them into a common list and put it in list.js file once everything is merged